cmd/agent/app/integrations.go

```diff
@@ -30,7 +30,7 @@ import (
 const (
 	reqAgentReleaseFile = "requirements-agent-release.txt"
 	constraintsFile     = "final_constraints.txt"
-	datadogPkgPattern   = "datadog-.*"
+	datadogPkgPattern   = "^datadog-[a-z0-9_-]*$"
```

Comment:
is the argument against which we apply this regex always "clean"? (no leading or trailing whitespaces or other similar characters?) If it's not clean this regex would fail.

Reply:
Oh actually, very good catch. This would have broken the install command, because we specify a version with `==`.

Reply:
hmm, this makes me realize that this may not be the best place to sanitize the module name... What do you think about putting the full sanitization logic in `getIntegrationName`?

I don't feel strongly about this, but I'd like the sanitization logic to be as close as possible to the string interpolation in the python code block